blueman/gui/manager/ManagerDeviceMenu.py

```diff
@@ -213,6 +250,34 @@ def Generate(self):
             self.append(item)
             return
 
+        # Generic (dis)connect
+        # LE devices do not appear to expose certain properties like uuids until connect to at least once.
+        # show generic connect for these as we will not show any way to connect otherwise.
+        device_uuids = self.SelectedDevice['UUIDs']
+        show_generic_connect = not device_uuids
+        for uuid in device_uuids:
+            if uuid128_to_uuid16(uuid) in (AUDIO_SOURCE_SVCLASS_ID, AUDIO_SINK_SVCLASS_ID,HANDSFREE_AGW_SVCLASS_ID,
+                                           HEADSET_SVCLASS_ID, HID_SVCLASS_ID):
+                show_generic_connect = True
+                break
+            elif int(uuid.split("-")[0][4:6], 16) == 0x18:
```

Comment:
After a deep dive in the BlueZ code and the Bluetooth Core Specification I have the following picture:

There seems to be no way to detect an LE device via the BlueZ API.
A GATT server with a GAP service is mandatory for LE devices.

We should check for the GAP service with our `GENERIC_ACCESS_SVCLASS_ID`.

I'm not sure what would happen for non-LE devices with a GAP service and non of the auto-connectable audio and input services but I don't expect such a device to exist and if it does it would be confusing anyway, so a connect option that does not have an effect would fit. 😄